src/state/opamState.ml

```diff
@@ -1265,6 +1265,11 @@ let string_of_cnf string_of_atom cnf =
 let string_of_conjunction string_of_atom c =
   Printf.sprintf "%s" (OpamStd.List.concat_map " , " string_of_atom (List.rev c))
 
+let string_of_stringlist l =
```

Comment:
it's not clear what this function does, if it's only useful locally, put it within the function that uses it.